packages/dds/tree/src/test/edit-manager/editManager.spec.ts

```diff
@@ -335,15 +380,15 @@ describe("EditManager", () => {
 			// Uncomment the code below to log the titles of generated scenarios.
 			// This is helpful for creating a unit test out of a generated scenario that fails.
 			// const title = scenario
-			//     .map((s) => {
-			//         if (s.type === "Pull") {
-			//             return `Pull(${s.seq}) from:${s.from} ref:${s.ref}`;
-			//         } else if (s.type === "Ack") {
-			//             return `Ack(${s.seq})`;
-			//         }
-			//         return `Push(${s.seq})`;
-			//     })
-			//     .join("|");
+			// 	.map((s) => {
+			// 		if (s.type === "Pull") {
+			// 			return `Pull(${s.seq}) from:${s.from} ref:${s.ref}`;
+			// 		} else if (s.type === "Ack") {
+			// 			return `Ack(${s.seq})`;
+			// 		}
+			// 		return `Push(${s.seq})`;
+			// 	})
+			// 	.join("|");
```

Comment:
Glad to see this was useful. :)

Reply:
It was as easy to use as it was indispensible for my debugging. Nice work :)